Dockerfile

```diff
@@ -2,37 +2,34 @@
 FROM python:3.9-slim-buster as builder
 
 # Set environment variables
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
 
 # Set work directory
 WORKDIR /server
 
-# Install system dependencies
-RUN apt-get update && apt-get install -y build-essential
-
-# Install Python dependencies
+# Install system dependencies and Python dependencies
 COPY ./server/requirements.txt /server/
-RUN pip wheel --no-cache-dir --no-deps --wheel-dir /server/wheels -r requirements.txt
+RUN apt-get update && apt-get install -y build-essential --no-install-recommends \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && pip wheel --no-cache-dir --no-deps --wheel-dir /server/wheels -r requirements.txt
 
 FROM python:3.9-slim-buster as runner
 
 WORKDIR /server
 
-# Install system dependencies
-RUN apt-get update && apt-get install -y netcat
-
-# Install Python dependencies
+# Install system dependencies and Python dependencies
 COPY --from=builder /server/wheels /server/wheels
 COPY --from=builder /server/requirements.txt .
-RUN pip install --no-cache /server/wheels/*
-RUN pip install uvicorn
+RUN apt-get update && apt-get install -y netcat --no-install-recommends \
```

Comment:
Is it necessary to install netcat? I don't think anything has a dependency on it.

Reply:
Probably not I kept this from the original dockerfile, will remove.